src/main/java/domain/calculator/vo/OperatorEnum.java

```diff
@@ -0,0 +1,34 @@
+package domain.calculator.vo;
+
+import domain.calculator.CalculatorAble;
+
+import java.util.function.BinaryOperator;
+
+public enum OperatorEnum {
+    PLUS("+", CalculatorAble::sum),
+    MINUS("-", CalculatorAble::subtract),
+    MULTIPLY("*", CalculatorAble::multiply),
+    DIVIDE("/", CalculatorAble::divide);
```

Comment:
요건 아래처럼 람다로 변경해볼 수 있지 않을까요? 예시 코드이니 실제 가능 여부는 코드 레벨에서 한번 확인해보시면 좋을 것 같아요.

```suggestion
public enum OperatorEnum {
    PLUS("+", a, b -> a + b),
    MINUS("-", a, b - > a - b),
    // ...
```

BinaryOperator